diff --git a/node_modules/@wordpress/api-fetch/build/utils/response.js b/node_modules/@wordpress/api-fetch/build/utils/response.js
index 832acf7..1093f19 100644
--- a/node_modules/@wordpress/api-fetch/build/utils/response.js
+++ b/node_modules/@wordpress/api-fetch/build/utils/response.js
@@ -47,10 +47,14 @@ const parseJsonAndNormalizeError = response => {
   };
 
   if (!response || !response.json) {
+		console.log(response);
+		console.log((new Error()).stack);
     throw invalidJsonError;
   }
 
   return response.json().catch(() => {
+		console.log(response);
+		console.log((new Error()).stack);
     throw invalidJsonError;
   });
 };
